packages/components/src/internal/components/search/FilterFacetedSelector.tsx

```diff
@@ -112,7 +112,7 @@ export const FilterFacetedSelector: FC<Props> = memo(props => {
                     </div>
                 )}
                 <Row>
-                    <Col xs={6}>
+                    <Col xs={taggedValues?.length > 0 ? 6 : 12}>
```

Comment:
Should the other Col below in the Row for tagged value also have similar conditional width check?